pkg/cloudprovider/suite_test.go

```diff
@@ -54,6 +56,90 @@ var _ = Describe("CloudProvider", func() {
 	})
 })
 
+var _ = Describe("Instance Type", func() {
+	It("should reserve additional allocatable memory for when HugePages is defined resources", func() {
```

Comment:
nit: I found this title a little misleading since it sound like it increased the allocatable memory. The memory that it's reserving is really from capacity, not allocatable, so I think it makes more sense like this. Let me know what you think though, I might have just read it weird.
```suggestion
	It("should reserve additional memory capacity for when HugePages is defined resources", func() {
```